lib/Backends/TraceEvents.cpp

```diff
@@ -64,8 +64,25 @@ void TraceContext::logTraceEvent(llvm::StringRef name, llvm::StringRef type,
   if (traceLevel_ == TraceLevel::NONE || traceLevel_ == TraceLevel::OPERATOR) {
```

Comment:
Can this function be implemented as `logTraceEvent(name, type, TraceEvent::now(), std::move(args)`? Or is getting the timestamp so expensive that it shouldn't be done before making sure the trace level warrants the creation of a `TraceEvent`?

Reply:
Getting timestamps is relatively cheap, I suppose.